### DISCUSSION TITLE
第一步安装 Docker  curl -sSL https://get.daocloud.io/docker | sh就跪了

### DISCUSSION THREAD

**Discussion by gogotomgogo** [MoeNetwork/Tieba-Cloud-Sign#228] [疑问]
![QQ截图20221223230142](https://user-images.githubusercontent.com/121304420/209359227-af123149-2385-4921-b5ff-a9c6c8774405.png)

请问这是为什么？

**Comment by gogotomgogo** [comment]
腾讯云       ubuntu 20.04 LTS

**Comment by n0099** [comment]
stdout都`>/dev/null`了谁知道apt-get在做什么？`-qq`参数也是同样的用意：https://serverfault.com/questions/644180/what-does-qq-argument-for-apt-get-mean
